swift/extractor/infra/SwiftDispatcher.h

```diff
@@ -263,9 +263,15 @@ class SwiftDispatcher {
 
   template <typename Tag, typename T, typename... Ts>
   bool fetchLabelFromUnionCase(const llvm::PointerUnion<Ts...> u, TrapLabel<Tag>& output) {
-    if (auto e = u.template dyn_cast<T>()) {
-      output = fetchLabel(e);
-      return true;
+    // we rely on the fact that when we extract `ASTNode` instances (which only happens
+    // on `BraceStmt` elements), we cannot encounter a standalone `TypeRepr` there, so we skip
+    // this case, which would be problematic as we would not be able to provide the corresponding
```

Comment:
I assume you mean:
```suggestion
    // this case; extracting `TypeRepr`s here would be problematic as we would not be able to provide the corresponding
```

Reply:
You might want to reflow that comment a bit 😄

Reply:
done :slightly_smiling_face: